examples/summarization/bart/README.md

```diff
@@ -14,6 +14,19 @@ python evaluate_cnn.py <path_to_test.source> cnn_test_summaries.txt
 ```
 the default batch size, 8, fits in 16GB GPU memory, but may need to be adjusted to fit your system.
 
+
+### Training
+Clone the repository and preprocess the data:
+```commandline
+git clone https://github.com/artmatsak/cnn-dailymail
```

Comment:
Can you make this a bash script as well. Include the install command for pytorch-lightning as they need the latest version and it is not included in the `requirements.txt` file. Also you will need to add a line to put the base class in the path (following https://github.com/huggingface/transformers/pull/3290)

Reply:
So you mean a bash script to preprocess the data? Since the user is downloading the dataset on their own, should we handle that by also downloading it for them as well in the bash script? And since the dataset is hosted on google drive, should we use `gdown` as a dependency? what are your thoughts?

Reply:
It's okay if you first tell them to download and preprocess the data separately. Just want to make sure the dependencies install and model run are in a script.